lib/register.dart

```diff
@@ -0,0 +1,306 @@
+import 'package:InstiComplaints/modals.dart';
+import 'package:flutter/material.dart';
+import 'dart:math';
+
+class BackgroundMaker extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height,
+        child: Stack(
+          fit: StackFit.expand,
+          children: <Widget>[
+            Container(
+              constraints: BoxConstraints.expand(),
+              color: Colors.white,
+              child: RegisterPage(),
+            ),
+            ClipPath(
+                clipper: CurveClipper(),
+                child: Container(
+                  constraints: BoxConstraints.expand(),
+                  color: Color(0xFF181D3D),
+                  child: Column(children: [
+                    SizedBox(height: MediaQuery.of(context).size.height / 8),
+                    Text(
+                      'Sign Up',
+                      style: Theme.of(context)
+                          .textTheme
+                          .headline5
+                          .apply(color: Colors.white),
+                    )
+                  ]),
+                )),
+          ],
+        ));
+  }
+}
+
+class CurveClipper extends CustomClipper<Path> {
+  @override
+  Path getClip(Size size) {
+    Path path = Path()
+      // set the "current point"
+      ..lineTo(0, size.width / 4)
+      ..addArc(
+          Rect.fromLTWH(0, size.width / 512, size.width / 2, size.width / 2),
+          pi,
+          -pi / 2)
+      ..lineTo(4 * size.width / 4, size.width / 2)
+      ..addArc(
+          Rect.fromLTWH(2 * size.width / 4, size.width / 2, size.width / 2,
```

Comment:
We produced a rectangle using the Rect.fromLTWH() function. The acronym LTWH stands for Left Top Width Height, which means we provide the top-left point, width and height to the function and it prepares the rectangle for us.